docs/sagemaker/source/reference/inference-toolkit.md

```diff
@@ -0,0 +1,141 @@
+# Inference Toolkit API
+
```

Comment:
Add github link of the toolkit https://github.com/aws/sagemaker-huggingface-inference-toolkit/tree/main